lib/chef/provider/route.rb

```diff
@@ -162,8 +162,10 @@ def action_delete
       def generate_config
         if platform_family?("rhel", "amazon", "fedora")
```

Comment:
tried fixing this to use the new helpers, it blew up unit tests, already burned too much time on this